service/service.py

```diff
@@ -140,6 +165,31 @@ def delete(self, rec_id):
         if recommendation:
             recommendation.delete()
         return '', status.HTTP_204_NO_CONTENT
+    
+    #------------------------------------------------------------------
+    # UPDATE AN EXISTING RECOMMENDATION
+    #------------------------------------------------------------------
+    @api.doc('update_recommendation', security='apikey')
+    @api.response(404, 'Recommendation not found')
+    @api.response(400, 'The posted Recommendation data was not valid')
+    @api.expect(recommendation_model)
+    @api.marshal_with(recommendation_model)
+    @token_required
+    def put(self, rec_id):
+        """
+        Update a Recommendation
+        This endpoint will update a Recommendation based the body that is posted
+        """
+        app.logger.info('Request to update recommendation with id: %s', rec_id)
+        check_content_type('application/json')
+        recommendation = Recommendation.find(rec_id)
+        if not recommendation:
+            raise NotFound("Recommendation with id '{}' was not found.".format(rec_id))
+        recommendation.deserialize(request.get_json())
```

Comment:
please refactor it according to https://github.com/nyu-devops/lab-flask-restplus-swagger/blob/master/service/service.py#L203
and make sure both the unit test and integration tests passed before sending out PRs